builder/Dockerfile.alpine-tracee-container

```diff
@@ -79,7 +79,8 @@ USER root
 RUN apk --no-cache update && \
     apk --no-cache add bash git rsync && \
     apk --no-cache add coreutils findutils && \
-    apk --no-cache add llvm clang go make gcc && \
+    apk --no-cache add llvm14 clang14 && \
```

Comment:
need to update alpine from 3.16 to 3.17 or we break the container build when attempting to install clang14